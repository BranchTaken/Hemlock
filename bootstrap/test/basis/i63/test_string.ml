open! Basis.Rudiments
open! Basis
open I63
open Format

let test () =
  let x = of_string "0" in
  printf "%a\n" pp x;
  let s = to_string (kv 1) in
  printf "%s\n" s

let _ = test ()
