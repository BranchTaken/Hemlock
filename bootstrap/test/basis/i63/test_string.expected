0i
1i
